test/throttle-binding-behavior.spec.js

```diff
@@ -143,7 +144,7 @@ describe('ThrottleBindingBehavior', () => {
       bindingEngine.observerLocator.eventManager,
       'mousemove',
       bindingEngine.parseExpression(`handleMouseMove($event) & throttle:${delay}`),
-      false,
+      delegationStrategy.none,
       false,
```

Comment:
Hi, it was already merged but I don't know how to update my fork. Can you help please? I tried but couldn't find any clue why.